src/main/webapp/app/exam/manage/exams/exam-checklist-component/exam-edit-workingtime-dialog/exam-edit-working-time-dialog.component.ts

```diff
@@ -0,0 +1,62 @@
+import { Component } from '@angular/core';
+import { NgbActiveModal } from '@ng-bootstrap/ng-bootstrap';
+import { faBan, faCheck, faSpinner } from '@fortawesome/free-solid-svg-icons';
+import { Exam } from 'app/entities/exam.model';
+import { EntityResponseType, ExamManagementService } from 'app/exam/manage/exam-management.service';
+
+@Component({
+    selector: 'jhi-edit-working-time-dialog',
+    templateUrl: './exam-edit-working-time-dialog.component.html',
+})
+export class ExamEditWorkingTimeDialogComponent {
+    exam: Exam;
+    examChange?: (exam: Exam) => void; // somehow, event emitter does not work
+
+    isLoading: boolean;
+
+    // used by *ngFor in the template
```

Comment:
What is used by *ngFor in the template? I don't see ngFor in the template of this component

Reply:
Copy paste, removed, thanks.